lib/src/platform_client.dart

```diff
@@ -217,7 +217,9 @@ class PlatformClient {
       {Position? position, String? message, Map<String, List<int>>? fileMap, bool? cannotTalk}) async {
     _verifyIsLoggedIn();
 
-    List<String> fileIds = await _sendPreCallMessage(message, fileMap);
+    if ((message?.isNotEmpty ?? false) || (fileMap?.isNotEmpty ?? false)) {
+      await _sendPreCallMessage(message, fileMap);
```

Comment:
`_sendPreCallMessage` throws an exception if the application does not support messaging.